frontend/packages/console-dynamic-plugin-sdk/src/runtime/plugin-loader.ts

```diff
@@ -5,7 +5,7 @@ import { PluginStore } from '@console/plugin-sdk/src/store';
 import { resolveEncodedCodeRefs } from '../coderefs/coderef-resolver';
 import { remoteEntryFile } from '../constants';
 import { ConsolePluginManifestJSON } from '../schema/plugin-manifest';
-import { overrideSharedModules } from '../shared-modules';
+import { overrideSharedModules } from '../shared-modules-override';
```

Comment:
FYI @rawagner @christianvogt in a follow-up PR, I'd like to remove the mechanism for plugin shared module overrides.

The existing `webpack.IgnorePlugin` used in `ConsoleRemotePlugin` that covers `@openshift-console/dynamic-plugin-sdk` module requests proves that we don't need this mechanic. At runtime, additional modules contributed by dynamic plugins (via module federation) become part of Console application's module registry, so it's safe to ignore them at plugin build time.

This will also reduce the generated plugin asset footprint, i.e. no more output files like
```
dist/vendors-node_modules_react_index_js-chunk.js
```